### PR TITLE
API-9770 Added glucose monitoring observation.

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

File: patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        "system" : "http://loinc.org",
+        "code" : "15074-8",
+        "display" : "Glucose-i"
+      }
+    ],
+    "text" : "Glucose-i"
+  },
+  "subject" : {
+    "reference" : "https://api.va.gov/services/fhir/v0/r4/Patient/1011537977V693883",
+    "display" : "LAB,TEST PATIENT FIVE"
```
Comment (jblefkove-va) [inline patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json:29]: This should match the display name for `1011537977V693883`

File: patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+  "resourceType" : "Observation",
+  "id" : "8cdc73e7-56e1-4823-b1ad-9b107c33d9a2",
+  "status" : "final",
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code" : "laboratory",
```
Comment (jblefkove-va) [inline patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json:10]: Is this hard-coded from DQ, or something that was present in the original sample?

File: patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  "subject" : {
+    "reference" : "https://api.va.gov/services/fhir/v0/r4/Patient/1011537977V693883",
+    "display" : "LAB,TEST PATIENT FIVE"
+  },
+  "effectiveDateTime" : "2009-10-15T01:39:00Z",
+  "issued" : "2009-10-15T02:00:30Z",
+  "performer" : [
+    {
+      "reference" : "Organization/700700:I",
```
Comment (jblefkove-va) [inline patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json:35]: fully qualified reference. Should be an I2 ID.

File: patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json (hunk 2 above)
Comment (taylorrassmannlibertyits) [reply patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json:10]: This was present in the original sample

Comment (taylorrassmannlibertyits) [reply patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json:10]: Apologies, this was NOT present.

Comment (jblefkove-va) [reply patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json:10]: Let's remove category then

File: patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  },
+  "subject" : {
+    "reference" : "https://api.va.gov/services/fhir/v0/r4/Patient/1011537977V693883",
+    "display" : "Ms. Carlita746 Kautzer186"
+  },
+  "effectiveDateTime" : "2009-10-15T01:39:00Z",
+  "issued" : "2009-10-15T02:00:30Z",
+  "performer" : [
+    {
+      "reference" : "https://api.va.gov/services/fhir/v0/r4/Observation/I2-72SYNKNS2L3HPXGWGHQ7NUPPLM000000",
```
Comment (jblefkove-va) [inline patient-generated-data-synthetic/src/test/resources/observation/8cdc73e7-56e1-4823-b1ad-9b107c33d9a2.json:35]: Wrong resource type?

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**